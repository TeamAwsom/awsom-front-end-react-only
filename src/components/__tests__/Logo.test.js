import React from 'react';
import Logo from '../Logo';
import renderer from 'react-test-renderer';

describe('Logo', () => {
  it('renders as expected', () => {
    const tree = renderer.create(<Logo />).toJSON();

    console.log(tree);
    expect(tree).toMatchSnapshot();
  });
});
