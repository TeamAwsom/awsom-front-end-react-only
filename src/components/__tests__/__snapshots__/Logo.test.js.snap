// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo renders as expected 1`] = `
<div
  className="logo"
>
  <img
    alt="awsom logo"
    src="OGAwsomLogo.png"
  />
</div>
`;
